lib/dynflow/persistence_adapters/sequel_migrations/021_add_unique_constraint_dynflow_actions_execution_plan_uuid_ukey.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+Sequel.migration do
+  up do
+    alter_table :dynflow_actions do
+      add_unique_constraint [:execution_plan_uuid], name: :dynflow_actions_execution_plan_uuid_ukey
```

Comment:
It definitely cannot be a unique constraint. There can be multiple actions belonging to a single execution plan and the tests are red because of this.